src/fiatconnect/slice.ts

```diff
@@ -53,13 +81,44 @@ export const slice = createSlice({
       state.quotesLoading = false
       state.quotesError = action.payload.error
     },
+    startFiatConnectTransfer: (state, action: PayloadAction<StartFiatConnectTransferAction>) => {
+      state.transfer = {
+        quoteId: action.payload.quoteId,
+        flow: action.payload.flow,
+        isSending: true,
+        failed: false,
+        txHash: null,
+      }
+    },
+    fiatConnectTransferFailed: (state, action: PayloadAction<FiatConnectTransferFailedAction>) => {
+      state.transfer = {
+        quoteId: action.payload.quoteId,
+        flow: CICOFlow.CashOut,
+        isSending: false,
+        failed: true,
+        txHash: null,
+      }
+    },
+    fiatConnectTransferSuccess: (
+      state,
+      action: PayloadAction<FiatConnectTransferSuccessAction>
+    ) => {
+      state.transfer = {
+        quoteId: action.payload.quoteId,
+        flow: CICOFlow.CashOut,
+        isSending: false,
+        failed: false,
+        txHash: action.payload.txHash,
+      }
+    },
   },
   extraReducers: (builder) => {
     builder.addCase(REHYDRATE, (state, action: RehydrateAction) => ({
       ...state,
       ...getRehydratePayload(action, 'fiatConnect'),
       quotesLoading: false,
       quotesError: null,
+      // transfer: null,
```

Comment:
Why is this here and commented out?

Reply:
oops, this should be uncommented. We'd want to reset/clean any in progress transfer state if the app reloads right?

Reply:
yep 👍